world.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+module.exports = function() {
+
+  var playgrounds = {},
+      notifyUpdate = function(){},
+      notifyDelete = function(){}
+
+  function tour() {
+    return Object.keys(playgrounds);
+  }
+
+  function onCodeObjectUpdate(fn) {
```

Comment:
Also, I understand why observability (`onUpdate`/`onDelete`) may be desirable in the near future, but it is dead code for now! Maybe try to use it in `app.js` to notify clients?